code/game/objects/effects/decals/cleanable/food.dm

```diff
@@ -41,6 +42,16 @@
 	if(is_species(AM, /datum/species/snail))
 		to_chat(AM, "<span class='danger'>Your path is obstructed by <span class='phobia'>salt</span>.</span>")
 
+/obj/effect/decal/cleanable/food/salt/Crossed(atom/movable/AM)
```

Comment:
tfw no parent call :(

Reply:
not sure that it needs it? was looking at other code snippets that override Crossed and they dont do it

Reply:
this and the other places probably should call parent given that there is a signal

https://github.com/tgstation/tgstation/blob/54dc879f192dc9c9de482c0eba8ca5b0b28b8d6b/code/game/atoms_movable.dm#L397-L398

Reply:
fixed